<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>485串口调试工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>超声波</string>
    </property>
    <widget class="QPushButton" name="btnUltrasonic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询超声波</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtUltrasonic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUltTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUltTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>陀螺仪</string>
    </property>
    <widget class="QPushButton" name="btnGyroscope">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询陀螺仪</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtGyroscope">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editGraTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGraTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>罗盘</string>
    </property>
    <widget class="QPushButton" name="btnCompass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询罗盘</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtCompass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnComTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editComTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>按键</string>
    </property>
    <widget class="QPushButton" name="btnButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询按键</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editButTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnButTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>红外</string>
    </property>
    <widget class="QPushButton" name="btnInfrared">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询红外</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtInfrared">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editInfTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInfTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>RFID</string>
    </property>
    <widget class="QPushButton" name="btnRFID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询RFID</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txtRFID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editRFIDTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRFIDTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开485串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="listPort">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="listBaud">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>@copyright谷歌科技2015</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>全局操作</string>
    </property>
    <widget class="QPushButton" name="btnALLTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editALLTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAllSensor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询所有传感器</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
